packages/patterns/src/constants.js

```diff
@@ -14,3 +19,9 @@ export const PATTERN_SYNC_TYPES = {
 	full: 'fully',
 	unsynced: 'unsynced',
 };
+
+// TODO: This should not be hardcoded. Maybe there should be a config and/or an UI.
+export const PARTIAL_SYNCING_SUPPORTED_BLOCKS = {
+	'core/paragraph': { content: __( 'Content' ) },
+	'core/button': { text: __( 'Text' ) },
```

Comment:
Should the image block src also be present?

Reply:
I think we can work on that later in a different PR as I think it might be more complicated than content 🤔.